cosmoz-moment.html

```diff
@@ -12,10 +31,10 @@
 		window.Cosmoz = window.Cosmoz || {};
 
 		var localSharedMoment = moment(),
```

Comment:
We can't remove it cause use it in `MomentBehavior` to set local moment variable:
 
    `Cosmoz.MomentBehavior = {
         properties: {
		locale: {
			type: String,
			readOnly: true,
			value: function () {
				return locale;
			}
		},
		moment: {
			type: Object,
			value: localSharedMoment
		}
	},`